### ISSUE TITLE
Copying a range will alter the worksheet names in a Formula

### ISSUE THREAD

**Issue by Floopy-Doo** [ClosedXML/ClosedXML#90]
I found an issue related to copying a range. 

When you copy a range to another cell and within this range there is an cell containing a formula that has cells within another worksheet the name is chaged.
For example, if the formula contains "='S10 Sample'!A4" and the range will be copied by 1 column to the right then the formula in the copied cell is "='T10 Sample'!B4". In change in the cell address is as expected but the change of the worksheet name is not expected and should not change. 

It would also be nice if there would be an option to not adjust the cell address when copying.

Here is also a code snippet: (The excel contains formulas within it cells)
```c#
FileInfo file = new FileInfo("SampleInput.xlsx");
using (FileStream stream = file.OpenRead())
{
    using (XLWorkbook workbook = new XLWorkbook(stream))
    {
        IXLWorksheet workSheet = workbook.Worksheet("S0 Summary");

        var cellRange = workSheet.Range("E1:B10");
        cellRange .Select();

        workSheet.Cell(1, 3).Value = cellRange 
 
        workbook.SaveAs("SampleOutput.xlsx");
    }
}
```

Thanks for any help in advance.

**Comment by igitur** [comment]
So there are 2 issues here:

1. You want the ability to copy a range without adjusting the formulas.
2. When you copy a range and the formulas are adjusted, there is a bug where the sheet name is also adjusted (probably some kind of bug in the regex).

Am I right?

**Comment by Floopy-Doo** [comment]
Yes you are right.

**Comment by igitur** [comment]
Ok, will look into it.

**Comment by igitur** [comment]
First part was solved in #109.
For the 2nd part, I recommend looping through the cells, copying the values and formulae separately.
